src/routes/+layout.svelte

```diff
@@ -212,11 +212,11 @@
       <QToolbarTitle>Quaff</QToolbarTitle>
       <QBtn
         icon={$Quaff.dark.isActive ? "light_mode" : "dark_mode"}
-        flat
+        design="flat"
```

Comment:
This seems to be an API change. Why?

Reply:
As I said for in the QBtn review comment, it's MD3 API rather than Quasar